### DISCUSSION TITLE
Feature Preview blue dot annoyance

### DISCUSSION THREAD

**Discussion by ghost** [community/community#48611] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Is there ANY way to disable this god forsaken blue dot notification in the top right? I don't care about feature previews and never want to see this.

**Comment by radar-blip** [comment]


**Reply by ghost** [reply]
...until the next time. Have you not used this?

**Reply by ghost** [reply]
Okay well you're useless and a troll so move on.

**Comment by melvincwng** [comment]


**Reply by ghost** [reply]
I take it you have no idea what I'm even talking about

**Comment by sbettwieser** [comment]
I wish I knew, myself. Every day I log in to find the blue dot is back and the command palette has re-enabled itself after I turned it off, for some reason.
